### PR TITLE
fixed Issue #787

### PR DESCRIPTION
**Pull request by Ali7040**
I just upgraded TypeScript using` yarn upgrade typescript@latest`. I didn't upgrade all dependencies because it caused test errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "resize-observer-polyfill": "^1.5.1",
     "tailwindcss": "^3.1.8",
     "ts-standard": "11.0.0",
```
Comment (vnugent) [inline package.json:118]: We want to upgrade ts-standard, not Typescript (unless it's required by ts-standard).

```
yarn upgrade ts-standard@^12.0.2
```

**Review by vnugent — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Ali7040) [reply package.json:118]: okay

Comment (Ali7040) [reply package.json:118]: image
@vnugent, if I upgrade to `ts-standard@^12.0.2`, we will need to make changes in a lot of files, and even after trying, I still get errors. What should I do?

Comment (vnugent) [reply package.json:118]: Some of them area valid warnings and we'll have to fix them one by one.  If you want  I can take over this PR and fix them.

Comment (Ali7040) [reply package.json:118]: I will try. Can you please tell me which area has valid warnings?

Comment (Ali7040) [reply package.json:118]: There's a problem:` <form onSubmit={handleSubmit(onSubmit)}>`. I am getting the error message Promise-returning function provided to the property where a void return was expected. in most of the files. However, I have solved other problems. Can you please guide me?

Comment (Ali7040) [reply package.json:118]: @vnugent we don't have eslint rules file at the root of the project.

Comment (Ali7040) [reply package.json:118]: image
@vnugent  Now only 8 warnings remain, and I have been trying to solve them, but I haven't found any solution yet.

**Comment by vnugent** [comment]
superseded by https://github.com/OpenBeta/open-tacos/pull/970

**Pull request closed without merging**